networking/configuring-networkpolicy.adoc

```diff
@@ -0,0 +1,16 @@
+[id="configuring-networkpolicy-plugin"]
+= Configuring network policy with OpenShift SDN
```

Comment:
Should "OpenShift SDN" be "OpenShiftSDN"?

Reply:
I've been pondering this myself. We have two related, but distinct items:

- `OpenShiftSDN` is the actual Kubernetes CNI plug-in
- OpenShift SDN is our software defined network, implemented by `OpenShiftSDN` plug-in

In 3.x, it was 3 distinct plug-ins, and they had names like `ovn-multitenant`, so it was easy to avoid conflating the terms.

Reply:
Hm. I would say that "`OpenShiftSDN`" is just a value that is used in certain config files, and "OpenShift SDN" is the actual name, for essentially all documentation purposes, because the documentation doesn't really need to distinguish "OpenShift SDN as our software defined network" from "OpenShift SDN as an actual Kubernetes CNI plug-in".

(It might be correct to say that "`openshift-sdn`" is the actual Kubernetes CNI plug-in, but I don't think that name ever gets used in the docs, and I don't think it needs to be.)